### DISCUSSION TITLE
Color in Date Picker

### DISCUSSION THREAD

**Discussion by henry5059** [AvaloniaUI/Avalonia#13108] [Q&A] [answered]
Hello, how can I define a color in specific days, months or years?
Since I would like to define a more background color or month as shown in the image.

![Captura de pantalla (27)](https://github.com/AvaloniaUI/Avalonia/assets/49210986/379def72-2c0d-401d-a1e4-adc27ad22ba1)

**Comment by robloo** [comment]
You may need to get into the details of `DatePickerPresenter` and make your own version of it.

Otherwise, if you dig into the control themes you might find a selector that can be used to match for specific values in the presenter. https://docs.avaloniaui.net/docs/next/reference/styles/style-selector-syntax#by-property-match

**Reply by henry5059** [reply, marked as answer]
Ohh interesting, okay, I'll check and thank you very much for the information.

**Comment by timunie** [comment]
I guess a custom style with a Converter for the Brushes may work.

**Reply by henry5059** [reply]
Okay, very much.
